visual-dotnet/SauceLabs.Visual/VisualCredential.cs

```diff
@@ -0,0 +1,34 @@
+using SauceLabs.Visual.Utils;
+
+namespace SauceLabs.Visual
+{
+    public class VisualCredential
```

Comment:
Since this is a public class available for customers, can we please add a description for each public method/property?